### ISSUE TITLE
BulkExecutor.InitializeAsync() broken at Azure.DocumentDB 2.1.2?

### ISSUE THREAD

**Issue by nikkostikis** [Azure/azure-cosmosdb-bulkexecutor-dotnet-getting-started#27]
The `bulkExecutor.InitializeAsync()` throws at Azure.DocumentDB 2.1.2. 
I tried it with 2.0.0 and it works fine. Was there something broken?
Does it fail to fetch the target Cosmos DB container partition map?

```
Exception thrown: 'System.MissingMethodException' in mscorlib.dll
iisexpress.exe Error: 0 : Error in BulkImportAsync: Method not found: 'System.Threading.Tasks.Task`1<Microsoft.Azure.Documents.Routing.CollectionRoutingMap> Microsoft.Azure.Documents.Routing.PartitionKeyRangeCache.TryLookupAsync(System.String, Microsoft.Azure.Documents.Routing.CollectionRoutingMap, System.Threading.CancellationToken)'.
```

**Comment by JordanLeg** [comment]
I have the same problem, you could solve it with the last Package?

**Comment by nikkostikis** [comment]
Nope. Using it with 2.0.0.

**Comment by JordanLeg** [comment]
I add more information about the error
![image](https://user-images.githubusercontent.com/44008737/47105795-8bb1ee80-d20a-11e8-99ad-bff6c40bdd46.png)

**Comment by rnagpal** [comment]
This issue has been fixed with the 1.1.2 release or higher of the BulkExecutor.
